torch/_inductor/codegen/wrapper.py

```diff
@@ -312,6 +300,21 @@ def __init__(self):
             self.write_get_cuda_stream
         )
 
+        @functools.lru_cache(None)
+        def add_import_once(line):
+            self.header.writeline(line)
+
+        self.add_import_once = add_import_once
```

Comment:
Where is add_import_once invoked?

Reply:
https://github.com/pytorch/pytorch/blob/786c697e4727848a4ab0597c18330fad786adf5e/torch/_inductor/select_algorithm.py#L286